README.md

```diff
@@ -512,18 +516,20 @@ MemorySession should be even faster, but has all the limitations documented abov
 Simply run
 
 ```bash
-pytest -k memory
+poetry run pytest -k memory
 ```
 
 or
 
 ```bash
-tox -- -k memory
+poetry run tox -- -k memory
 ```
 
 ## Releasing new versions
 
-- Bump version in `sqs_workers/__version__.py`
+- Bump version in `pyproject.toml`
 - Update the CHANGELOG
 - Commit the changes with a commit message "Version X.X.X"
-- Push the changes to GitHub and PyPI with a single command `make upload`
+- Tag the current commit with `vX.X.X`
+- Push the changes to GitHub with `git push origin vX.X.X`
```

Comment:
More like: "create a release on GitHub named vX.X.X". It's a bit nicer, IMO.

Reply:
Ah ok makes sense 👍